.travis.yml

```diff
@@ -1,6 +1,17 @@
 sudo: false
 language: groovy
```

Comment:
I think you can refactor this according to the below pattern. Have you tried that, perchance?
```
language:
  - python
  - groovy
```